design/networking-blog-post.md

```diff
@@ -0,0 +1,73 @@
+# A New Network Stack for Zcash
+
+In our [2020 Engineering Roadmap][roadmap], we gave an overview of our plans
+for Zebra.  Announced last summer at Zcon1, Zebra aims to support the core
+strength of Zcash – its best-in-class cryptography – by placing it on a solid
+foundation, providing a modern, modular implementation that can be broken into
+components and used in many different contexts.  In that post, we briefly
+described the new network stack we designed and implemented for Zebra.  As a
+fork of Bitcoin, Zcash inherited Bitcoin's network protocol; in this post,
+we'll do a deep dive on Zebra's network stack.
+
+The new design is based on service-oriented architecture concepts from the
```

Comment:
I've generally heard this term ("service-oriented architecture") used to refer to the idea of decoupling an application into services over the network — it's basically an Enterprise-y or more outdated way of saying "microservices" https://en.wikipedia.org/wiki/Service-oriented_architecture

I'm not sure if I'd call `tower` a "service oriented architecture" library. 

BTW, one of the main inspirations behind Tower was [this paper](https://monkey.org/~marius/funsrv.pdf), which Sean also mentions in his hyper 0.13 [announcement post](https://seanmonstar.com/post/189594157852/hyper-v013). I'm not sure if that's useful to reference here or not but I thought it might be worth mentioning.